frontend/database/modules/datasets.js

```diff
@@ -818,10 +818,28 @@ const actions = {
     /**
      * Fetch all observation datasets from backend
      */
-
-    return await ObservationDataset.api().get("/datasets/", {
+    const previousDatasets = await ObservationDataset.api().get("/datasets/", {
       persistBy: "create",
     });
+
+    const feedbackTaskDatasets = await this.$axios.get("v1/datasets");
+    const formattedDatasetsObj = feedbackTaskDatasets.data.map(
+      (feedbackTask) => {
+        return {
+          ...feedbackTask,
+          task: "FeedbackTask",
+          created_at: feedbackTask.inserted_at,
+          last_updated: feedbackTask.updated_at,
+          workspace: feedbackTask.id,
```

Comment:
We're assigning the `dataset.id` to the workspace but should be the dataset id. Please, correct me if I'm wrong.
```suggestion
          id: feedbackTask.id,
```